src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private Bulldozer bulldozer1 = new Bulldozer();
+    private Bulldozer bulldozer2 = new Bulldozer();
+    private Bulldozer bulldozer3 = new Bulldozer();

```

Comment:
```suggestion
    private Bulldozer firstBulldozer = new Bulldozer();
    private Bulldozer secondBulldozer = new Bulldozer();
    private Bulldozer thirdBulldozer = new Bulldozer();
```
the same for other implementations of MachineProducer